docs/getting-started.md

```diff
@@ -10,7 +10,7 @@ For general information about Distbench, please consult the
 
 ### Dependencies
 
-To build Distbench, you need to have Bazel installed.
+To build Distbench, you need to have Bazel and g++11.3 installed.
```

Comment:
Due to homa ? We should fix the Kokoro test scripts too; not sure if 11.3 is available there...